e2e/deployers/subscription.go

```diff
@@ -0,0 +1,85 @@
+package deployers
+
+import (
+	"github.com/ramendr/ramen/e2e/util"
+	"github.com/ramendr/ramen/e2e/workloads"
+)
+
+type Subscription struct {
+	NamePrefix string
+	McsbName   string
+
+	ChannelName      string
+	ChannelNamespace string
+}
+
+func (s *Subscription) Init() {
+	s.NamePrefix = "sub-"
+	s.McsbName = "default"
+	s.ChannelName = "ramen-gitops"
```

Comment:
(future) these (channel name/namespace) should come from some global configuration in the future. We would need to use the same tools in "other" environments where channel and policy details are already configured. For now we can assume these as the defaults.

Reply:
ok. I moved channel name and namespace to util/const.go file.  

Kesler also implemented code to first get such value from env variables, if they exist. Will add that later.